services/data_services.py

```diff
@@ -23,18 +23,17 @@ def unbson(chunk):
 client = MongoClient('mongodb')
 
 
-# def get_current_questionnaire(urlname):
-#     questionnaires = client.dsm.questionnaires
-#
-#     quest_bson = questionnaires.find_one({
-#         'urlname': urlname,
-#         'version.major': 1,
-#         'deleted_on': None
-#     }, sort=[("version.minor", DESCENDING)])
-#
-#     if quest_bson != None:
-#         quest = unbson(quest_bson)
-#     return quest
+def get_current_questionnaire(urlname):
+    questionnaires = client.dsm.questionnaires
+
+    quest_bson = questionnaires.find_one({
+        'urlname': urlname,
+        'deleted_on': None
+    }, sort=[("version.minor", DESCENDING)])
+
+    if quest_bson != None:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The `==` and `!=` operators use the compared objects&#39; ` __eq__` method to test if they are equal. To check if an object is a singleton, such as `None`, we recommend that you use the `is` identity comparison operator.

[Learn more](https://docs.python.org/3/reference/expressions.html#is)